docs/source/guide/index.rst

```diff
@@ -134,204 +133,233 @@ Step 1: Configure the Roles
-load the keys.
+    ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+    ┃                                   STEP 2: Load the Online Key                                    ┃
+    ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
 
-.. code:: shell
 
-    ╔══════════════════════════════════════════════════════════════════════════════════════════════════╗
-    ║                                     STEP 2: Load roles keys                                      ║
-    ╚══════════════════════════════════════════════════════════════════════════════════════════════════╝
+                                            The Online Key
 
-    The keys must have a password and the file must be accessible.
+    The online key is the same one provided to the Repository Service for TUF Workers (RSTUF Worker).
+    This key is responsible for signing the snapshot, timestamp, targets, and delegated targets (hash
+    bin) roles.
 
-    Depending on the Organization, each key has an owner. During the key loading process, it is
-    important that the owner of the key inserts the password.
+    The RSTUF Worker uses this key during the process of managing the metadata.
 
-    The password or the key content is not shown on the screen.
+    Note: the ceremony process won't show any password or key content.
 
-    Ready to start loading the keys? Passwords will be required for keys [y/n]: y
+    Choose 1/1 ONLINE key type [ed25519/ecdsa/rsa] (ed25519):
+    Enter 1/1 the ONLINE`s private key path: tests/files/online.key
+    Enter 1/1 the ONLINE`s private key password:
+    ✅ Key 1/1 Verified
 
-    Enter 1/2 the root`s Key path: tests/files/JanisJoplin.key
-    Enter 1/2 the root`s Key password:
-    ✅ Key 1/2 Verified
+    Step 3: Validate the information/settings
+    .........................................
 
-    Enter 2/2 the root`s Key path: tests/files/JimiHendrix.key
-    Enter 2/2 the root`s Key password:
-    ✅ Key 2/2 Verified
+    After confirming all details, the initial payload for bootstrap will be
+    complete (without the offline keys).
 
-    Enter 1/2 the targets`s Key path: tests/files/KurtCobain.key
-    Enter 1/2 the targets`s Key password:
-    ✅ Key 1/2 Verified
 
-    Enter 2/2 the targets`s Key path: tests/files/ChrisCornel.key
-    Enter 2/2 the targets`s Key password:
-    ✅ Key 2/2 Verified
+Step 3: Load Root Keys
+......................
 
-    Enter 1/1 the snapshot`s Key path: tests/files/snapshot1.key
-    Enter 1/1 the snapshot`s Key password:
-    ✅ Key 1/1 Verified
+It is essential to define the key owners. There is a suggestion in the CLI.
 
-    Enter 1/1 the timestamp`s Key path: tests/files/timestamp1.key
-    Enter 1/1 the timestamp`s Key password:
-    ✅ Key 1/1 Verified
+The owners will need to be present to insert their keys and use their passwords
+to load their keys.
```

Comment:
I just noticed we mention `their` three times in a row now.
Maybe we can do something like:
```suggestion
The owners will need to be present to insert their keys and passwords
to load their keys.
```